torch/package/_mock.py

```diff
@@ -1,23 +1,84 @@
-
-_magic_methods = ['__subclasscheck__', '__hex__', '__rmul__', '__complex__',
```

Comment:
Can you keep the old formatting?

Reply:
there are the clang suggested formatted changes

Reply:
I would still undo the formatting changes. Also, this code isn't technically part of TorchScript, so you may not even need to modify it.

Reply:
removed!